stream_chat/client.py

```diff
@@ -321,6 +321,11 @@ def update_message_partial(
     def delete_message(self, message_id: str, **options: Any) -> StreamResponse:
         return self.delete(f"messages/{message_id}", options)
 
+    def undelete_message(self, message_id: str, user_id: str) -> StreamResponse:
+        return self.post(
+            f"messages/{message_id}/undelete", data={"undeleted_by": user_id}
+        )
```

Comment:
Should we add this to async client as well? :)

Reply:
Good one. I updated it now 👍